Makefile

```diff
@@ -6,16 +6,12 @@ PYMOR_PYTEST_MARKER?=None
 all:
 	./dependencies.py
 
-# PyPI wants ReStructured text
-README.rst: README.md
-	pandoc -f markdown_github -t rst $< > $@
-
 # I want HTML (to preview the formatting :))
 README.html: README.md
-	pandoc -f markdown_github -t html $< > $@
+	pandoc -f gfm -t html $< > $@
```

Comment:
Hmpf, I guess on my pandoc this won't work. Do you know in which version they changed this? (And is there any good reason why they broke backwards compatibility?)

Reply:
Ok, so is the change really needed ATM? I have pandoc 1.17, which won't change before buster.

Reply:
Can you check if your pandoc actually needs the ```-f``` flag? Mine converts the markdown input to html without it while producing no diff to a run where I specified ```-f markdown_github```.

Reply:
On my machine there's a diff ..

Reply:
https://github.com/pymor/pymor/pull/518/commits/2e9f444f7f291ee2e61fb4d52c936f6c9234d823